pkg/syncer/metadatasyncer.go

```diff
@@ -305,6 +315,11 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 				return logger.LogNewErrorf(log, "failed to start CnsVolumeOperationRequest informer on %q instances. Error: %v",
 					cnsvolumeoperationrequest.CRDSingular, err)
 			}
+			err = startStoragePolicyQuotaCRInformer(ctx, k8sConfig)
```

Comment:
I think we need to check the PodVMOnStretchedSupervisor feature state, similar to here:
https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/2667/files#diff-8708c209224e511b429f0d0da8b658577d5d708614676e6399c45aa69c2e7a97R294

Reply:
the code is under "if isPodVMOnStretchSupervisorFSSEnabled {" check above, and isPodVMOnStretchSupervisorFSSEnabled is initialized in same function, just above pkg/syncer/metadatasyncer.go:L234